### PR TITLE
Add documentation for new Valve integration

### PR DESCRIPTION
**Pull request by cibernox**
## Proposed change
Documents the upcoming new Valve integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
Most of the wording is copied from the Cover integration, as it's very similar although with less features.

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/102184
- Link to parent pull request in the Brands repository: Doesn't apply (I think)
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by jbouwh** [comment]
@cibernox you need to resolve a merge conflict here...

**Comment by cibernox** [comment]
@jbouwh done!

File: source/_integrations/valve.markdown [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+You can change the kind of valve you are representing in the [customize section](/docs/configuration/customizing-devices/). The following device classes are supported for valves:
+
+- **None**: Generic valve. This is the default and doesn't need to be set.
+- **water**: Valve that controls the flow of water through a system.
+- **gas**: Valve that controls the flow of gas through a system.
+
+## Services
+
+### Cover control services
```
Comment (MartinHjelmare) [inline source/_integrations/valve.markdown]:
```suggestion
### Valve control services
```

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_integrations/valve.markdown [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  - Valve
+ha_release: 2024.1.0
+ha_quality_scale: internal
+ha_codeowners:
+  - '@home-assistant/core'
+ha_domain: valve
+ha_integration_type: entity
+---
+
+Home Assistant can give you an interface to control valves like water, gas or air valves.
```
Comment (c0ffeeca7) [inline source/_integrations/valve.markdown]:
```suggestion
Home Assistant can give you an interface to control valves such as water, gas, or air valves.
```

File: source/_integrations/valve.markdown [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+{% include integrations/building_block_integration.md %}
+
+## Device Class
```
Comment (c0ffeeca7) [inline source/_integrations/valve.markdown]:
```suggestion
## Device class
```

File: source/_integrations/valve.markdown [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+- **None**: Generic valve. This is the default and doesn't need to be set.
+- **water**: Valve that controls the flow of water through a system.
+- **gas**: Valve that controls the flow of gas through a system.
+
+## Services
+
+### Cover control services
+
+All valves respond to `valve.open`, `valve.close` and `valve.toggle`.
```
Comment (c0ffeeca7) [inline source/_integrations/valve.markdown]:
```suggestion
All valves respond to `valve.open`, `valve.close`, and `valve.toggle`.
```

File: source/_integrations/valve.markdown [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+Valves that allow intermediate position may be controlled also with `valve.set_position` and `valve.stop`.
```
Comment (c0ffeeca7) [inline source/_integrations/valve.markdown]:
```suggestion
Valves that allow for an intermediate position may also be controlled with `valve.set_position` and `valve.stop`.
```

File: source/_integrations/valve.markdown [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+ha_integration_type: entity
+---
+
+Home Assistant can give you an interface to control valves like water, gas or air valves.
+
+{% include integrations/building_block_integration.md %}
+
+## Device Class
+
+You can change the kind of valve you are representing in the [customize section](/docs/configuration/customizing-devices/). The following device classes are supported for valves:
```
Comment (c0ffeeca7) [inline source/_integrations/valve.markdown]:
```suggestion
You can change the device class of the valve in the [customize section](/docs/configuration/customizing-devices/). Valves support the following device classes:
```

File: source/_data/glossary.yml [hunk 7]
```diff
@@ -387,6 +387,14 @@
   definition: >-
     TTS (text-to-speech) allows Home Assistant to talk to you.
   link: /integrations/tts/
+
+
+- term: Valve
+  definition: >-
+    Valves are devices the control the flow of liquid and gasses. They all can be opened
```
Comment (c0ffeeca7) [inline source/_data/glossary.yml]:
```suggestion
    Valves are devices to control the flow of liquids and gases. All valves in Home Assistant can be opened
```

File: source/_data/glossary.yml [hunk 8]
```diff
@@ -387,6 +387,14 @@
...
+    and closed and some can also be set to a specific position.
```
Comment (c0ffeeca7) [inline source/_data/glossary.yml]:
```suggestion
    and closed. Some valves can also be set to a specific position.
```

File: source/_integrations/valve.markdown [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+  - Valve
+ha_release: 2024.1.0
+ha_quality_scale: internal
+ha_codeowners:
+  - '@home-assistant/core'
+ha_domain: valve
+ha_integration_type: entity
+---
+
+Home Assistant can give you an interface to control valves such as water, gas, or air valves.
```
Comment (c0ffeeca7) [inline source/_integrations/valve.markdown]:
```suggestion
The valve entity in Home Assistant provides an interface to control valves such as water, gas, or air valves.
```

File: source/_integrations/valve.markdown [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+- **None**: Generic valve. This is the default and doesn't need to be set.
+- **water**: Valve that controls the flow of water through a system.
+- **gas**: Valve that controls the flow of gas through a system.
+
+## Services
+
+### Valve control services
+
+All valves respond to `valve.open`, `valve.close`, and `valve.toggle`.
+Valves that allow for an intermediate position may also be controlled with `valve.set_position` and `valve.stop`.
```
Comment (c0ffeeca7) [inline source/_integrations/valve.markdown]:
```suggestion
Valves that allow setting a specific position may also be controlled with `valve.set_position` and `valve.stop`.
```
suggesting to use "specific" here as this is the term you used in the glossary.

File: source/_integrations/valve.markdown [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+    at: "07:15:00"
+  action:
+    - service: valve.close
+      target:
+        entity_id: valve.demo
+```
+
+### Service `valve.set_position`
+
+Set valve position of one or multiple valves, if they support intermediate positions.
```
Comment (c0ffeeca7) [inline source/_integrations/valve.markdown]:
```suggestion
Set valve position of one or multiple valves, if they support intermediate positions.
```
```suggestion
Set the position of one or multiple valves if they support setting a specific position.
```

File: source/_integrations/valve.markdown [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | yes | String or list of strings that point at `entity_id`'s of valves. Use `entity_id: all` to target all.
+| `position` | no | Integer between 0 and 100.
```
Comment (c0ffeeca7) [inline source/_integrations/valve.markdown]:
```suggestion
| `position` | no | Integer between 0 and 100.
```
What do the values mean? 0 means closed, 100 means the valve is open?

Comment (cibernox) [reply source/_integrations/valve.markdown]: Yes. I clarified it.

File: source/_integrations/valve.markdown [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+```
+
+### Service `valve.set_position`
+
+Set the position of one or multiple valves if they support setting a specific position.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | yes | String or list of strings that point at `entity_id`'s of valves. Use `entity_id: all` to target all.
+| `position` | no | Integer between 0 (fully closed), and 100 (fully open).
```
Comment (c0ffeeca7) [inline source/_integrations/valve.markdown]:
```suggestion
| `position` | no | Integer between 0 (fully closed) and 100 (fully open).
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @cibernox :+1:

**Pull request merged**